fix: add missing config for base network

platykurtic-icu opened the pull request:
### What I did

I included the apparently missing config for Base environment.

fixes: #2 

### How I did it

### How to verify it

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
- [x] New test cases have been added and are passing
- [ ] ~Documentation has been updated~
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

Commit:
add missing config for base network

Commit:
fix: add missing config for base network

Fix a missing config for Base network, and also reorder alphabetically the networks to make a new commit title :)

Commit:
test: include test case for the ecosystem configs

antazoey commented on tests/test_blockscout.py:
```diff
@@ -123,3 +123,11 @@ def test_get_contract_type_with_rate_limiting(mock_backend, explorer):
+
+@pytest.mark.parametrize("ecosystem", ["base", "ethereum", "gnosis", "optimism", "polygon"])
+def test_config_ecosystem_exists(ecosystem):
+    from ape import config
```
this could probably just go at the top of the module

antazoey commented on tests/test_blockscout.py:
```diff
@@ -123,3 +123,11 @@ def test_get_contract_type_with_rate_limiting(mock_backend, explorer):
     assert "BLOCKSCOUT_API_KEY" in actual
+
+
+@pytest.mark.parametrize("ecosystem", ["base", "ethereum", "gnosis", "optimism", "polygon"])
```
for a more dynamic test against future additions, we could import and use keys from https://github.com/ApeWorX/ape-blockscout/blob/main/ape_blockscout/utils.py#L11

antazoey reviewed (approved):
thanks! good catch.
see feedback for improvements if you have a chance

Commit:
fix: resolved PR comments

- Moved import to top of module
- Made test case more dynamic by using the NETWORKS object exported from the module
